h/sql_tasks/tasks/report/create_from_scratch/03_annotation_group_counts/01_create_view.sql

```diff
@@ -2,22 +2,12 @@ DROP MATERIALIZED VIEW IF EXISTS report.annotation_group_counts CASCADE;
 
 CREATE MATERIALIZED VIEW report.annotation_group_counts AS (
     SELECT
-        data.*,
-        EXTRACT('year' FROM created_week - INTERVAL '6 month')::smallint AS academic_year,
-        CONCAT(
-            EXTRACT('year' FROM created_week - INTERVAL '6 month'),
-            '-',
-            CASE WHEN EXTRACT('quarter' FROM created_week - INTERVAL '6 month') < 3 THEN 1 ELSE 2 END
-        ) AS academic_half_year,
```

Comment:
We're going to be slicing and dicing this data in so many ways, we'll probably need more and less than this depending on situation. We can do better with functions elsewhere.